diff --git a/liboboe/oboe_api.hpp b/liboboe/oboe_api.hpp
index c2ef015f4..391adad11 100644
--- a/liboboe/oboe_api.hpp
+++ b/liboboe/oboe_api.hpp
@@ -59,6 +59,7 @@ class Metadata : private oboe_metadata_t {
     static Metadata *makeRandom(bool sampled = true);
     static Metadata *fromString(std::string s);
 
+    void padZeros();
     oboe_metadata_t *metadata();
 
 #ifdef SWIGJAVA
diff --git a/liboboe/oboe_api.hpp b/liboboe/oboe_api.hpp
index c2ef015f4..391adad11 100644
--- a/liboboe/oboe_api.hpp
+++ b/liboboe/oboe_api.hpp
@@ -59,6 +59,7 @@ class Metadata : private oboe_metadata_t {
     static Metadata *makeRandom(bool sampled = true);
     static Metadata *fromString(std::string s);
 
+    void padZeros();
     oboe_metadata_t *metadata();
 
 #ifdef SWIGJAVA
